news/make.rst

```diff
@@ -0,0 +1,23 @@
+**Added:**
+
+* <news item>
+
+**Changed:**
+
+* <news item>
+
+**Deprecated:**
+
+* <news item>
+
+**Removed:**
+
+* <news item>
+
+**Fixed:**
+
+* added missing make to requirements/docs.txt
```

Comment:
For consistency per our standards:

"Add ``make`` to ``requirements/docs.txt`` for Windows."

?

Reply:
I pushed a fix